actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -27,6 +27,7 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.request_id_header = ActionDispatch::Constants::X_REQUEST_ID
     config.action_dispatch.log_rescued_responses = true
     config.action_dispatch.debug_exception_log_level = :fatal
+    config.action_dispatch.domain_extractor = ActionDispatch::Http::URL::DomainExtractor
```

Comment:
This will require this class to be loaded at this point. It would be fine but I prefer to delay it to the configuration like I suggested.

Reply:
Or alternatively, you could check if the config is set, and if not, just don't set it to anything and the default in the `mattr_accessor` would work

Reply:
Fixed. Can you review it again plz?